### ISSUE TITLE
Front end data entry

### ISSUE THREAD

**Issue by doowrednu** [trilbymedia/grav-plugin-flex-objects#15]
Is it possible to use front end Forms to add records to the json data ?

**Comment by mahagr** [comment]
Yes it is! I will need to add some instructions on how to do that.

**Comment by PeterB513** [comment]
I am struggeling to create a frontend data entry form to add objects to a flex directory. Can you give some hints how to create this. The learn enviroment says that it is possible but not how. Thanks.
